examples/python_sdk/README.md

```diff
@@ -69,3 +85,41 @@ You can then run the `using_connections_config.py` file to print search results:
 ```
 python using_connections_config.py
 ```
+
+### 3. Executing concurrent queries with throttling
+
+`concurrent-example.py` is an example for running concurrent search
+requests on your cortex search service along with throttling the
+request rate when the server is busy.
+
+This example collects connection parameters from a file.  Add the
+following lines to a file named `.env` in the current directory:
+
+```
+SNOWFLAKE_ACCOUNT=AID123456
+SNOWFLAKE_USER=myself
+SNOWFLAKE_PASSWORD=pass123...
```

Comment:
We should ensure there is flexibility for users who can't use username-password auth and must use `authenticator: "externalbrowser"`. This is the case eg for the PM account, which requires OAuth for login. https://docs.snowflake.com/en/developer-guide/python-connector/python-connector-api#label-snowflake-connector-methods-connect

Reply:
Something like the following:

```
 load_dotenv("./.env")

    connection_params = {
        "account": os.getenv("SNOWFLAKE_ACCOUNT"),
        "user": os.getenv("SNOWFLAKE_USER"),
        "database": os.getenv("SNOWFLAKE_DATABASE"),
        "schema": os.getenv("SNOWFLAKE_SCHEMA"),
    }
    password = os.getenv("SNOWFLAKE_PASSWORD", None)
    authenticator = os.getenv("SNOWFLAKE_AUTHENTICATOR", None)
    
    if password is not None:
        connection_params["password"] = password
    if authenticator is not None:
        connection_params["authenticator"] = authenticator

    session = Session.builder.configs(connection_params).create()
    
    root = Root(session)
    search_service = root.databases[os.getenv("SNOWFLAKE_DATABASE")].schemas[os.getenv("SNOWFLAKE_SCHEMA")].cortex_search_services[os.getenv("SNOWFLAKE_CORTEX_SEARCH_SERVICE")]
   ```

Reply:
done.